cmd/ztest/ztest.c

```diff
@@ -6526,6 +6528,12 @@ ztest_resume_thread(void *arg)
 		 */
 		if (ztest_random(10) == 0)
 			zfs_abd_scatter_enabled = ztest_random(2);
+
+		/*
+		 * Periodically inject remapping delays (10% of the time).
+		 */
+		zfs_object_remap_one_indirect_delay_ms =
+		    ztest_random(10) == 0 ? ztest_random(1000) : 0;
```

Comment:
I would give this some base value that will always be there when injection is turned on. This way we dont end up "injecting" 0ms of delay

Reply:
Sure, I can add a `+ 1` we're always injecting in the range of 1-1000.

Reply:
I meant something more like `ztest_random(10) == 0 ? 200 + ztest_random(800) : 0;`
Just so that injection always has meaningful impact.

Reply:
I think there's value is testing the entire range of plausible delays.

Reply:
ok. fair enough.